template/templates/addons/kubernetes-dashboard/app/helmrelease.yaml.j2

```diff
@@ -31,24 +31,32 @@ spec:
   values:
     env:
       TZ: "${TIMEZONE}"
-    extraArgs:
-      - --enable-skip-login
-      - --disable-settings-authorizer
-      - --enable-insecure-login
-      - --token-ttl=43200
-    ingress:
-      enabled: true
-      className: nginx
-      annotations:
-        cert-manager.io/cluster-issuer: letsencrypt-staging
-        hajimari.io/icon: mdi:kubernetes
-      hosts:
-        - &host "kubernetes.${SECRET_DOMAIN}"
-      tls:
-        - hosts:
-            - *host
-          secretName: kubernetes-dashboard-tls
+    api:
+      containers:
+        args:
+          - --enable-insecure-login=true
+          - --enable-skip-login=true
+          - --disable-settings-authorizer=true
+    app:
+      ingress:
+        enabled: true
+        className: nginx
+        annotations:
+          cert-manager.io/cluster-issuer: letsencrypt-staging
```

Comment:
Should this by default be staging, should this be here at all?

Reply:
It should be removed due to using a wildcard cert

Reply:
Done